bsp/esp32_p4_function_ev_board/esp32_p4_function_ev_board.c

```diff
@@ -393,8 +387,9 @@ esp_err_t bsp_touch_new(const bsp_touch_config_t *config, esp_lcd_touch_handle_t
         },
     };
     esp_lcd_panel_io_handle_t tp_io_handle = NULL;
-    const esp_lcd_panel_io_i2c_config_t tp_io_config = ESP_LCD_TOUCH_IO_I2C_GT911_CONFIG();
-    ESP_RETURN_ON_ERROR(esp_lcd_new_panel_io_i2c((esp_lcd_i2c_bus_handle_t)BSP_I2C_NUM, &tp_io_config, &tp_io_handle), TAG, "");
+    esp_lcd_panel_io_i2c_config_t tp_io_config = ESP_LCD_TOUCH_IO_I2C_GT911_CONFIG();
+    tp_io_config.scl_speed_hz = CONFIG_BSP_I2C_CLK_SPEED_HZ;
+    ESP_RETURN_ON_ERROR(esp_lcd_new_panel_io_i2c(bsp_i2c_handle, &tp_io_config, &tp_io_handle), TAG, "");
```

Comment:
Here should be used function `esp_lcd_new_panel_io_i2c_v2`

Reply:
@espzav Actually, the correct v1 or v2 call is selected at build time here https://github.com/espressif/esp-idf/blob/master/components/esp_lcd/include/esp_lcd_io_i2c.h#L118

The `_Generic` keyword select v1 or v2 based on passed type https://www.geeksforgeeks.org/_generic-keyword-c/

Reply:
Ok, thank you for correct me. Then it is ok.